dotplot/index.js

```diff
@@ -86,135 +100,138 @@ d3.selectAll('.framed')
     });
 
 parseData.load(dataURL, { sort, sortOn })
-.then(({ groupNames, plotData, valueExtent, data }) => { // eslint-disable-line no-unused-vars
+    .then(({
+        groupNames, plotData, valueExtent, data,
```

Comment:
` 104:44  error  'data' is defined but never used  no-unused-vars`

Please remove extraneous `data` arg if unused, or move the `// eslint-disable-line no-unused-vars` line from below up to this line.